src/views/Accounts/Create.vue

```diff
@@ -1,140 +1,325 @@
+      'activeNetwork',
+      'enabledAssets'
+    ]),
+    ...mapGetters(['networkAssets']),
+    chains () {
+      return buildConfig.chains
+        .map((chainId) => {
+          const { name, code, nativeAsset } = chains[chainId]
+          return {
+            id: chainId,
+            name,
+            code,
+            nativeAsset
+          }
+        })
+        .filter((chain) => chain.id !== this.selectedChain?.id)
+    },
+    accountIndex () {
+      const _accounts =
+        this.accounts[this.activeWalletId]?.[this.activeNetwork].filter(
+          (a) => a.chain === this.selectedChain?.id
+        ) || []
+      if (_accounts.length <= 0) {
+        return 0
+      }
+
+      const lastAccount = _accounts.sort((a, b) => {
+        if (a.index > b.index) {
+          return -1
+        }
+
+        return 0
+      })[0]
+
+      return lastAccount.index + 1
+    },
+    accountName () {
+      return `${this.selectedChain?.name} ${this.accountIndex + 1}`
+    },
+    inputsValidated () {
+      return this.selectedChain &&
+          this.accountAlias &&
+          this.accountAlias.length > 5 &&
+          this.accountColor &&
+          this.accountColor.length > 5
     }
   },
   created () {
-    this.selectedAsset = this.assetList[0]
+    let chain
+    if (this.chainId) {
+      chain = this.chains.find((c) => c.id === this.chainId)
+    }
+    this.selectedChain = chain || Object.values(chains)[0]
+    this.accountColor = getNextAccountColor(
+      this.selectedChain.id,
+      this.accountIndex - 1
+    )
   },
   methods: {
+    ...mapActions([
+      'createAccount',
+      'getUnusedAddresses',
+      'updateAccountBalance'
+    ]),
     getAssetIcon,
-    goToOverview () {
-      this.$router.replace('/wallet')
+    cancel () {
+      this.$router.back()
     },
-    selectAsset (asset) {
-      this.selectedAsset = asset
+    selectChain (chain) {
+      this.selectedChain = chain
     },
     toggleAssetList () {
       this.assetsDropdownOpen = !this.assetsDropdownOpen
     },
     hideAssetList () {
       this.assetsDropdownOpen = false
     },
-    createAccount () {
-      // TODO: implement create account
+    async createNewAccount () {
+      this.loading = true
+      const assetKeys =
+        this.enabledAssets[this.activeNetwork]?.[this.activeWalletId] || []
+
+      const assets = assetKeys.filter((asset) => {
+        return cryptoassets[asset].chain === this.selectedChain.id
+      })
+      const account = {
+        name: this.accountName,
+        alias: this.accountAlias,
+        chain: this.selectedChain.id,
+        addresses: [],
+        assets,
+        index: this.accountIndex,
+        type: 'default',
+        enabled: true,
+        color: this.accountColor
+      }
+      const createdAccount = await this.createAccount({
+        network: this.activeNetwork,
+        walletId: this.activeWalletId,
+        account
+      })
+
+      await this.getUnusedAddresses({
```

Comment:
Why not put the updating addresses and balances inside the `createAccount` action?

Reply:
fixed, now calling required actions inside `createAcction` action